integration/ServeHeader.go

```diff
@@ -0,0 +1,156 @@
+package integration
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"reflect"
+	"strconv"
+
+	"github.com/onsi/gomega/types"
+	"github.com/paketo-buildpacks/occam"
+)
+
+// ServeHeader matches if the actual occam.Container is running AND the
+// response from an HTTP request to the container's exposed port
+// contains a header with the expected value
+func ServeHeader(expectedName, expectedValue string) *ServeMatcher {
```

Comment:
Why is this needed over what you added in https://github.com/paketo-buildpacks/occam/pull/169?